book/content/software_checklist/01/making_software_citable.md

```diff
@@ -0,0 +1,67 @@
+# Making software citable
+
+Digital Object Identifiers are globally unique identifiers which can point to
+any digital object, such as a version of a paper, a version of software etc.
+This has the advantage that it is unambigous and standardized. For papers, using
+DOIs is commonplace, and a DOI is usually provided by the publisher. For
+software, you can make your own DOI with [Zenodo](https://zenodo.org/):
+
+1. You can tell people how to cite your software by including a ``CITATION.cff``
```

Comment:
I don't understand the use of double backticks here, usually names of files are in single backticks in markdown

Reply:
this line was removed